tamr_unify_client/project/attributemappingcollectionwithoutbase.py

```diff
@@ -0,0 +1,33 @@
+import requests
+
+from tamr_unify_client import Client
+from tamr_unify_client.auth import UsernamePasswordAuth
+from tamr_unify_client.project.attributemappingwithoutbase import AttributeMappingNoBase
+
+
+class AttributeMappingCollectionNoBase(AttributeMappingNoBase):
+    def __init__(self, map_url):
+        self.map_url = map_url
+        self.all_maps = requests.get(self.map_url).json()
+        self.map_list = []
+        x = 0
+        for mapping in self.all_maps:
+            self.map_list.append(self.all_maps[x])
+        x += 1
+
+    def setUp(self):
+        auth = UsernamePasswordAuth("username", "password")
+        self.unify = Client(auth)
+
+    def by_resource_id(self, resource_id):
```

Comment:
This is creating a concept of an ID by index, which isn't present in the API. The ID wouldn't be stable, because as mappings are created and deleted the order they appear in is not guaranteed to remain the same. The resource ID of a mapping with relative ID `projects/1/attributeMappings/3551-5` should be `3551-5`. Also, open for discussion, but I think it would make sense to get the mappings in the collection at streaming time instead of creation time to be consistent with other collections.